cmd/node.go

```diff
@@ -141,7 +141,7 @@ func (node *Node) Start(exitChannels ...*chan struct{}) {
 
 	// Setup chain database
 	dbDir := lib.GetBadgerDbPath(node.Config.DataDirectory)
-	opts := lib.PerformanceBadgerOptions(dbDir)
+	opts := badger.DefaultOptions(dbDir)
```

Comment:
This is a significant change that reduces memory by 30-40 GB. But also comes at tradeoffs of smaller Badger transactions. Could you blocksync with this change and see if everything works?